src/frontend/efiling-frontend/src/domain/package/package-review/DocumentList.js

```diff
@@ -57,6 +58,18 @@ export default function DocumentList({
 
   return (
     <div className="ct-document-list">
+      {showDownloadAlert && (
+        <Toast
+          content="Something went wrong while trying to download your document"
+          setShow={setShowDownloadAlert}
+        />
+      )}
+      {showWithdrawAlert && (
+        <Toast
+          content="Something went wrong while trying to withdraw your document"
+          setShow={setShowWithdrawAlert}
+        />
+      )}
```

Comment:
Out of curiosity, what happens if there is more than one error that needs to display at the same time?  Ie, the document list contains more than one item, the network goes down, and the user clicks all the links one after the other.

Reply:
I answered my own question.  Looks like the errors stack.
Well done!